aten/src/ATen/native/quantized/cpu/qupsample_nearest2d.cpp

```diff
@@ -192,6 +223,15 @@ Tensor upsample_nearest2d_quantized_cpu(
   return _upsample_nearest2d_quantized_cpu<nearest_neighbor_compute_source_index>(input, osize, scale_h, scale_w);
 }
 
+Tensor& upsample_nearest2d_quantized_out_cpu(
+    const Tensor& input,
+    IntArrayRef osize,
+    c10::optional<double> scale_h,
+    c10::optional<double> scale_w,
+    Tensor& output) {
+  return _upsample_nearest2d_quantized_out_cpu<nearest_neighbor_compute_source_index>(input, osize, scale_h, scale_w, output);
```

Comment:
I think `_upsample_nearest2d_quantized_out_cpu` is not templated? and the argument is also different, it's take a list of scale_factors, instead of scale_h and scale_w.
Do you know why do we have this discrepancy?